docs/releases.md

```diff
@@ -1,4 +1,79 @@
-## Branch strategy
+# Release Process
+
+#### 1. Maintainers quorum
+
+* A majority of maintainers must agree on a single commit SHA for the release. The agreement should be documented in the release notes.
+* The maintainers will match the commit SHA to the release `major.minor.patch` in the release notes
+* The maintainers will agree on a changelog proposal that MUST be in a PR prior to the release.
+* The main `REAMDE.md` must also reflect the new release. Ideally this should be in the same PR.
+
+#### 2. Merge the release notes
+
+* `HISTORY.md` serves as our official changelog.
+* Before the release, the maintainers had opened a PR with the proposed changelog. This should now be merged.
+
+
+#### 3. Release kops to github
+
+*  Darwin binary (generated)
+*  Linux binary (generated)
+*  Source code (zip)
+*  Source code (tar.gz)
```

Comment:
Need to list:

- ami
- nodeup in s3
- kops in s3
- protokube docker
- dnscontroller docker
- channels <- this needs to pull a tag, not master